peps/pep-0790.rst

```diff
@@ -0,0 +1,154 @@
+PEP: 790
```

Comment:
You can use PEP 790 if you like. However 758 is the next available, if that's what you were after.

```console
❯ pip install -q pepotron
❯ pep next
Next available PEP: 758
```

```suggestion
PEP: 758
```